### PR TITLE
Make the sender name appear in the bold weight font, fix overflow issues

### PR DESCRIPTION
**Pull request by shawnborton**
This makes the message sender name appear in the correct bold weight font, and fixes overflow issues in the message sender header as well as the sidebar footer when usernames are super long.

### Fixed Issues
N/A, but relevant discussion here: https://expensify.slack.com/archives/C03U7DCU4/p1608314618489300

### Tests
1. Open up a chat and start sending messages, make sure the sender name appears in bold
2. Sign up with a super long email address and make sure it is truncated properly in the Left Hand Navigation footer, and make sure the sender line with Sender Name + Timestamp breaks to two lines on smaller viewports (mobile).

### Screenshots
image

image

image

image

### CODE REVIEW

**Comment by shawnborton** [comment]
Reassigning PullerBear since Bondy is OOO this week.

**Comment by Julesssss** [comment]
Verified on Desktop, Web, iOS, Android, mobile Web 👍

File: src/pages/home/sidebar/SidebarBottom.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ const SidebarBottom = ({myPersonalDetails, network, insets}) => {
             </View>
             <View style={[styles.flexColumn]}>
                 {myPersonalDetails.displayName && (
-                    <Text style={[styles.sidebarFooterUsername]}>
+                    <Text style={[styles.sidebarFooterUsername]} numberOfLines={1}>
```
Comment (Julesssss) [inline src/pages/home/sidebar/SidebarBottom.js:64]: Are we allowed to have multiple props in a single line?

Comment (Julesssss) [reply src/pages/home/sidebar/SidebarBottom.js:64]: Yes we are.

**Review by Julesssss — APPROVED** [review]

**Pull request merged**